charts/hub/templates/embeddings-secret.yaml

```
@@ -0,0 +1,15 @@
+{{- if and .Values.embeddings.enabled (not .Values.embeddings.auth.existingSecret) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "hub.embeddingsSecretName" . }}
+  labels:
+    {{- include "hub.labels" . | nindent 4 }}
+    app.kubernetes.io/component: embeddings
+type: Opaque
+data:
+  {{ .Values.embeddings.auth.secretKey | default "EMBEDDING_PROVIDER_API_KEY" }}: {{ include "hub.embeddingsApiKey" . | b64enc }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)